docs/cn/datasets/icdar2015_CN.md

```diff
@@ -8,8 +8,29 @@ ICDAR 2015 [文章](https://rrc.cvc.uab.es/?ch=4)
 <details>
   <summary>从何处下载 ICDAR 2015</summary>
 
+ICDAR 2015 挑战赛分为三个任务。任务1是文本定位。任务3是单词识别。任务4是端到端文本检测识别。任务2文本分割的数据不可用。
+
+### Text Localization
+
+这四个文件分别是 ch4_training_images.zip、ch4_training_localization_transcription_gt.zip、ch4_test_images.zip 和 Challenge4_Test_Task1_GT.zip。它们的下载链接如下所示。
+
+
+Training Set
+
+[Training Set Images (88.5MB)](https://rrc.cvc.uab.es/?com=downloads&action=download&ch=4&f=aHR0cHM6Ly9ycmMuY3ZjLnVhYi5lcy8/Y29tPWRvd25sb2FkcyZhY3Rpb249ZG93bmxvYWQmZmlsZT1jaDRfdHJhaW5pbmdfaW1hZ2VzLnppcA==).- 1000 images obtained with wearable cameras
```

Comment:
This link contains your individual credentials and will not work on other machines. You need to replace it with the common download link: https://rrc.cvc.uab.es/?ch=4&com=downloads
The same applies to all datasets listed here.

Reply:
I will correct them. Thanks!